src/Compilers/CSharp/Portable/Symbols/Source/SourceAssemblySymbol.cs

```diff
@@ -2357,6 +2357,10 @@ private void DecodeWellKnownAttribute(ref DecodeWellKnownAttributeArguments<Attr
 
                 arguments.GetOrCreateData<CommonAssemblyWellKnownAttributeData>().AssemblyAlgorithmIdAttributeSetting = algorithmId;
             }
+            else if (attribute.IsTargetAttribute(this, AttributeDescription.NullablePublicOnlyAttribute))
+            {
+                ReportExplicitUseOfNullabilityAttribute(in arguments, AttributeDescription.NullablePublicOnlyAttribute);
```

Comment:
>ReportExplicitUseOfNullabilityAttribute(in arguments, AttributeDescription.NullablePublicOnlyAttribute); [](start = 16, length = 104)

Isn't this attribute a module level attribute? I do not think we should duplicate what the attribute usage attribute is for.

Reply:
This was intended for `SourceModuleSymbol`. Moved.

---
In reply to: [296890713](https://github.com/dotnet/roslyn/pull/36152#discussion_r296890713) [](ancestors = 296890713)